src/components/BlockHomeIntro.vue

```diff
@@ -57,6 +57,9 @@ const modules = [Pagination];
 .intro {
   padding: 20px 0;
   width: 100%;
+  padding: 45px;
+  height: calc(100% - $FooterHeight - $HeaderHeight);
+  // footer height: 127px; header height: 80px
```

Comment:
remove the commit

Reply:
what? is something wrong here?

Reply:
remove   // footer height: 127px; header height: 80px